### PR TITLE
Raise player on cover click and Ctrl+click from panel

### PR DESCRIPTION
**Pull request by luisbocanegra**
Closes: #14

### CODE REVIEW

**Comment by ccatterina** [comment]
Hi @luisbocanegra , in these days I'm really busy. I'll try to review the PR in the weekend! Ty

File: src/contents/ui/main.qml [hunk 1]
```diff
@@ -141,6 +165,38 @@ PlasmoidItem {
...
+                                anchors.centerIn: parent
+                                width: raisePlayerTooltip.width
+                                height: raisePlayerTooltip.height
+                                propagateComposedEvents: true
+                                cursorShape: Qt.PointingHandCursor
+                            }
+                        }
+                        visible: coverMouseArea.containsMouse
+                        delay: 0
+                    }
```
Comment (ccatterina) [inline src/contents/ui/main.qml:199]: I prefer the standard PlasmaComponents3.Tooltip, both for the look and consistency with the other tooltips, something like this should be fine:
```qml
PlasmaComponents3.ToolTip {
      id: raisePlayerTooltip
      anchors.centerIn: parent
      text: player.canRaise ? i18n("Bring player to the front") : i18n("This player can't be raised")
      visible: coverMouseArea.containsMouse
}
```

**Review by ccatterina — CHANGES_REQUESTED** [review]

File: src/contents/ui/main.qml (hunk 1 above)
Comment (luisbocanegra) [reply src/contents/ui/main.qml]: Done

**Review by ccatterina — APPROVED** [review]

**Comment by ccatterina** [comment]
Thank you @luisbocanegra! :rocket:

**Pull request merged**